.github/ISSUE_TEMPLATE/general_issue.md

```diff
@@ -0,0 +1,32 @@
+---
+name: General issue
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please fill out the sections below to help us address your issue.
+
+### Version of AWS SDK for PHP?
+
+
+### Version of AWS Service Provider for Laravel 5?
```

Comment:
Version 1.x of the service provider works for Laravel 4, so I'd remove the `5` here.